### PR TITLE
[IT-2268] Fix managed policies

### PR DESCRIPTION
**Pull request by jesusaurus**
We need to use different parameters for managed policies managed by AWS vs managed by us. Update to use a template with support for the parameter used by custom managed policies.

depends-on: https://github.com/Sage-Bionetworks/aws-infra/pull/360

### CODE REVIEW

**Review by xschildw — APPROVED** [review]

File: org-formation/600-access/_tasks.yaml [hunk 1]
```diff
@@ -181,25 +181,27 @@ CostExplorerAccessPolicy:
...
-        }
-      ]
-    }'
+    PolicyDocument:
+      !Sub
+        '{
+          "Version": "2012-10-17",
+          "Statement": [
+            {
+              "Sid": "${policyCostExplorerSid}",
```
Comment (zaro0508) [inline org-formation/600-access/_tasks.yaml:190]: where is `policyCostExplorerSid` defined for this reference?

File: org-formation/700-aws-sso/_tasks.yaml [hunk 2]
```diff
@@ -278,7 +278,7 @@ SsoDeveloper:
       - 'arn:aws:iam::aws:policy/PowerUserAccess'
       - 'arn:aws:iam::aws:policy/AdministratorAccess-AWSElasticBeanstalk'
       - 'arn:aws:iam::aws:policy/AWSBillingReadOnlyAccess'
-      - !CopyValue [ !Sub '${primaryRegion}-cost-explorer-access-policy-ManagedPolicyArn' ]
+    customerManagedPolicyName: !Ref policyCostExplorerSid
```
Comment (zaro0508) [inline org-formation/700-aws-sso/_tasks.yaml:281]: i think the reference needs to be the policy name?  I don't think a policy's Sid is the same as a policy's name.  You probably need to output the policy name in the template that created the policy then reference it here.

**Review by zaro0508 — CHANGES_REQUESTED** [review]

File: org-formation/600-access/_tasks.yaml (hunk 1 above)
Comment (jesusaurus) [reply org-formation/600-access/_tasks.yaml:190]: I'm adding it to `_parameters.yaml`: https://github.com/Sage-Bionetworks-IT/organizations-infra/pull/616/files#diff-69957a1b2df5fa15bc2b9dc2bb4c2321375dcf5966a15b43600657e1c34c7f12R24

File: org-formation/700-aws-sso/_tasks.yaml (hunk 2 above)
Comment (jesusaurus) [reply org-formation/700-aws-sso/_tasks.yaml:281]: Oh, hrm, I was interpreting that as the name that AWS uses, not the name that CloudFormation uses. I wonder which it is

File: org-formation/600-access/_tasks.yaml (hunk 1 above)
Comment (zaro0508) [reply org-formation/600-access/_tasks.yaml:190]: ahh, ok.  have you tested this with `org-formation print-task`?

File: org-formation/700-aws-sso/_tasks.yaml (hunk 2 above)
Comment (zaro0508) [reply org-formation/700-aws-sso/_tasks.yaml:281]: I believe the Sid is just a description of the policy

Comment (jesusaurus) [reply org-formation/700-aws-sso/_tasks.yaml:281]: After some testing I found that we need to set the ManagedPolicyName parameter in the template and use that value. I've updated the dependent PR to also add support for that parameter as well.

File: org-formation/600-access/_tasks.yaml (hunk 1 above)
Comment (jesusaurus) [reply org-formation/600-access/_tasks.yaml]: yeah, `npm run print-tasks-failfast` is happy

**Review by zaro0508 — CHANGES_REQUESTED** [review]

File: org-formation/_parameters.yaml [hunk 3]
```diff
@@ -20,3 +20,7 @@ allRegions:
 AdminCentralCfnBucket:
   Type: String
   Default: bootstrap-awss3cloudformationbucket-19qromfd235z9
+
+CostExplorerPolicyName:
+  Type: String
+  Default: CostExplorerAccess
```
[the remark this answers is not in the document]
Comment (jesusaurus) [reply org-formation/_parameters.yaml:26]: yeah, I can do that, that name is much less likely to be duplicated

File: org-formation/600-access/_tasks.yaml [hunk 4]
```diff
@@ -181,25 +181,26 @@ CostExplorerAccessPolicy:
     Account: !Ref MasterAccount
```
[the remark this answers is not in the document]
Comment (jesusaurus) [reply org-formation/600-access/_tasks.yaml:181]: whoops, yeah it does

**Review by zaro0508 — APPROVED** [review]

**Pull request merged**